lnwallet/chanfunding/coin_select.go

```diff
@@ -52,11 +52,68 @@ func selectInputs(amt btcutil.Amount, coins []Coin) (btcutil.Amount, []Coin, err
 	return 0, nil, &ErrInsufficientFunds{amt, satSelected}
 }
 
+// calculateFees returns for the specified utxos and fee rate two fee
+// estimates, one calculated using a change output and one without. The weight
+// added to the estimator from a change output is for a P2WKH output.
+func calculateFees(utxos []Coin, feeRate chainfee.SatPerKWeight) (
+	btcutil.Amount, btcutil.Amount, error,
```

Comment:
weird line wrapping

Reply:
I wasn't sure if that was a formatting rule or not (as per https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#48-additional-style-constraints-on-top-of-gofmt). I will change it to not have the wrapping placed there.